packages/react/src/patterns/sub-patterns/ButtonGroup/ButtonGroup.js

```diff
@@ -31,6 +31,7 @@ const ButtonGroup = ({ buttons }) => (
       return (
         <li key={key} className={`${prefix}--buttongroup-item`}>
           <Button
+            tabIndex={key === 0 ? 2 : 1}
```

Comment:
If it's primary, shouldn't it be a lower tab index than higher?

Reply:
> If it's primary, shouldn't it be a lower tab index than higher?

@jeffchew The QA guys and Wonil confirmed they want the secondary button focused first because visually it's before the primary button

Reply:
I'm curious why tabindex needs to be set at all then if it looks to be the natural tab order now?

Reply:
The natural tab order focuses on primary first

Reply:
Is this the way the ButtonGroup is coded then?

Reply:
Yeah i agree, me and @RaphaelAmadeu were talking about this earlier

Reply:
@andysherman2121 @RaphaelAmadeu @annawen1 
Going to approve for now, but one thought I have is that we might want to come back and readdress how the props are pulled in. My initial thought is to have a separate prop for a single primary button, and another prop for an array of secondary buttons.